buildozer/targets/android.py

```diff
@@ -897,6 +897,12 @@ def execute_build_package(self, build_cmd):
             cmd.append('--depend')
             cmd.append(gradle_dependency)
 
+        # support for manifestPlaceholders
+        manifest_placeholders = self.buildozer.config.getdefault('app', 'android.manifest_placeholders', '[:]')
+        if manifest_placeholders:
```

Comment:
Just thinking, given the default is `[:]` (above) we would always enter this condition and always end up with that `--manifest-placeholders` flag from now on. Is that fine? Do we really need that `if` in that case?

Reply:
It makes sense if the user for some unknown reason decides to do something like that into his `buildozer.spec` file:
`android.manifest_placeholders = `